modules/test/playwright/utils/performLogin.ts

```diff
@@ -60,7 +66,7 @@ async function performLogin(
 	}
 	else {
 		await page
-			.getByLabel('Sign In- Loading')
+			.locator('.login-container')
```

Comment:
Mmm, not sure about this change. I remember having seen some modifications from other developer to cover a different use case. Maybe this breaks that fix :thinking: 

I'm referring to the latest three commits for this file -> https://github.com/liferay/liferay-portal/commits/master/modules/test/playwright/utils/performLogin.ts

Reply:
I can revert this bit, I thought it minor in importance. I simply don't like when `getByLabel` includes parts that are not visible in UI. Some ugliness in needed in either case as there are two `Sign In` buttons in UI (main page and within modal). Maybe there is something here that I'm not seeing, maybe there is a case when there is no `login-container`, so I'll revert it.

Reply:
Reverted!